pkg/component/seedsystem/seedsystem.go

```diff
@@ -47,7 +46,7 @@ type Values struct {
 // ReserveExcessCapacityValues contains configuration for the deployment of the excess capacity reservation resources.
 type ReserveExcessCapacityValues struct {
 	// Enabled specifies whether excess capacity reservation should be enabled.
-	Enabled bool
+	Enabled *bool
```

Comment:
Suggestion: Return to non-pointer and use the `SeedSettingExcessCapacityReservationEnabled` function to set this field. This avoids duplication for logic `pointer.BoolDeref(s.values.ReserveExcessCapacity.Enabled, true)`.